projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -455,32 +455,17 @@ public function add_jetpack_menu() {
 	}
 
 	/**
-	 * Re-adds the Site Editor menu without the (beta) tag, and where we want it.
+	 * Update Site Editor menu item's link and position.
 	 */
-	public function add_gutenberg_menus() {
+	public function update_gutenberg_menus() {
 		// We can bail if we don't meet the conditions of the Site Editor.
 		if ( ! ( function_exists( 'gutenberg_is_fse_theme' ) && gutenberg_is_fse_theme() ) ) {
```

Comment:
We should remove these checks as well. They were only there because originally Nav Unification operated by wiping out all the menus and then re-adding everything and we needed that logic.

The menu item will only have been added under Gutenberg's own logic and won't be touched if it isn't found. This will make Automattic/wp-calypso#54620 able to run without extra work, also.